src/fontra/views/editor/panel-reference-font.js

```diff
@@ -305,6 +305,20 @@ export default class ReferenceFontPanel extends Panel {
         this.requestReferenceFontsPreview();
       }
     );
+
+    editorController.sceneSettingsController.addKeyListener("location", () => {
+      const fontVariationSettings = [];
+      for (const axis of this.editorController.fontController.globalAxes) {
+        fontVariationSettings.push(
+          `'${axis.tag}' ${this.editorController.sceneSettings.location[axis.tag]}`
```

Comment:
Did you test this? Because it doesn't work unless you apply this:
```suggestion
          `'${axis.tag}' ${this.editorController.sceneSettings.location[axis.name]}`
```

Reply:
Ah, it does work when looking at MutatorSans.ttf (where axis.name == axis.tag) but not with MutatorSans.designspace (where axis.name != axis.tag).